pkg/services/policer/policer.go

```diff
@@ -65,6 +65,14 @@ type Option func(*cfg)
 // the redundant local copy of the object.
 type RedundantCopyCallback func(oid.Address)
 
+// Network provides information about the NeoFS network to Policer for work.
+type Network interface {
+	// IsLocalNodeInNetmap checks whether the local node belongs to the current
+	// network map. If it is impossible to check this fact, IsLocalNodeInNetmap
+	// returns false.
+	IsLocalNodeInNetmap() bool
```

Comment:
`IsOnMap`? `IsOnline`? Although this one works as well.

Reply:
Depends on interface name btw.